lab_1/start.py

```diff
@@ -2,17 +2,35 @@
 Concordance implementation starter
 """
 
-from main import read_from_file
 import os
+import main
 
 if __name__ == '__main__':
-    #  use data.txt file to test your program
+    #  here goes your logic: calling methods from concordance.py
     current_dir = os.path.dirname(os.path.abspath(__file__))
-    data = read_from_file(os.path.join(current_dir, 'data.txt'))
-    stop_words = []
+    data = main.read_from_file(os.path.join(current_dir, 'data.txt'))
+    stop_words = main.read_from_file(os.path.join(current_dir, 'stop_words.txt'))
 
-    #  here goes your logic: calling methods from concordance.py
+    #  use data.txt file to test your program
+    tokens = main.tokenize(data)
+    print('tokenize', tokens[:5])
+
+    tokens = main.remove_stop_words(tokens, stop_words)
+    print('remove_stop_words', tokens[:5])
+
+    frequencies = main.calculate_frequencies(tokens[:5000])
+    print('calculate_frequencies', frequencies[tokens[0]])
+
+    concordance = main.get_concordance(tokens, 'dog', 2, 0)
+    print('get_concordance', concordance[:5])
+
+    adjacent = main.get_adjacent_words(tokens, 'dog', 2, 0)
+    print('get_adjacent_words', adjacent[:5])
+
+    sorted_concordance = main.sort_concordance(tokens, 'dog', 2, 0, True)
+    print('sort_concordance', sorted_concordance[:5])
+
+    main.write_to_file('', sorted_concordance)
 
-    RESULT = None
-    # DO NOT REMOVE NEXT LINE - KEEP IT INTENTIONALLY LAST
-    assert RESULT == [(), ()], 'Concordance not working'
+    RESULT = sorted_concordance
+    assert RESULT, 'Concordance not working'

```

Comment:
assert RESULT == [expected result]
'
you should put in `[]` what you expect, for example, `[['the', 'man'], ['we', 'are']]`